recipe/meta.yaml

```diff
@@ -20,14 +20,14 @@ requirements:
   host:
     - python {{ python_min }}
     - pip
-    - setuptools >=64.0.0
-    - setuptools-scm >1.5.4
+    - setuptools >=64
+    - setuptools-scm >8
   run:
     - python >={{ python_min }}
     - asciitree
     - numpy >=1.24
     - fasteners
-    - numcodecs >=0.10.0,!=0.14.0,!=0.14.1
+    - numcodecs >=0.10.0,!=0.14.0,!=0.14.1,<0.16.0a0
```

Comment:
Adding the upperbound to Numcodecs to address issue: https://github.com/zarr-developers/zarr-python/issues/2963

Reply:
Also added a repodata patch for older Zarr 2 versions with PR: https://github.com/conda-forge/conda-forge-repodata-patches-feedstock/pull/1009